coral-trino/src/test/resources/hive-from-trino.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+
+    Copyright 2019 LinkedIn Corporation. All rights reserved.
+    Licensed under the BSD-2 Clause license.
+    See LICENSE in the project root for license information.
+
+-->
+<configuration>
+  <property>
+    <name>hive.exec.scratchdir</name>
+    <value>${coral.from.trino.test.dir}/hive-scratch-dir</value>
+  </property>
+  <property>
```

Comment:
Do we need to copy this file?

Reply:
Not needed, using just `hive.xml `in `ToRelTestUtils` and it's working fine.

Reply:
So are you planning to delete this file?